src/views/document/utils/document-view-mixin.js

```diff
@@ -67,6 +68,12 @@ export default {
         },
       ];
     },
+    meta: function () {
+      if (!this.document) {
+        return null;
+      }
+      return this.documentOpenGraph();
```

Comment:
```suggestion
      if (this.document) {
        return this.documentOpenGraph();
      }
      return null;
```

Reply:
I'd rather keep the logic the other way: first return fast with excluding conditions, then expand the main logic (although it's simple here)